paddle/fluid/framework/transfer_scope_cache.cc

```diff
@@ -17,12 +17,61 @@
 namespace paddle {
 namespace framework {
 
+#ifdef PADDLE_WITH_MKLDNN
+using transfer_data_cache_map = std::unordered_map<size_t, Scope*>;
+using transfer_scope_cache_map = std::unordered_set<Scope*>;
+static std::unordered_map<size_t, transfer_data_cache_map*>
+    static_transfer_data_caches;
+static std::unordered_map<size_t, transfer_scope_cache_map*>
+    static_transfer_scope_caches;
+#endif
+
 std::unordered_map<size_t, Scope*>& global_transfer_data_cache() {
+#ifdef PADDLE_WITH_MKLDNN
+  size_t tid = static_cast<size_t>(platform::get_cur_thread_id());
```

Comment:
get_cur_thread_id->get_cur_mkldnn_session_id

Reply:
new PR #18453 is submitted

Reply:
solved